<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RCallControl</class>
 <widget class="QWidget" name="RCallControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>34</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="AVControl" name="wgtAVControl" native="true">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>86</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="CustomLabel" name="lblStatus">
     <property name="text">
      <string notr="true">lblStatus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAccept">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnHangup">
     <property name="text">
      <string>Hangup</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header location="global">utils/customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>AVControl</class>
   <extends>QWidget</extends>
   <header>avcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
